examples/fluids/qfunctions/blasius.h

```diff
@@ -320,11 +287,11 @@ CEED_QFUNCTION(Blasius_Inflow_Jacobian)(void *ctx, CeedInt Q,
                                 q_data_sur[3][i]
                                };
 
-    const CeedScalar u_normal = Dot3(norm, velocity);
+    const CeedScalar u_normal = Dot3(norm, s.Y.velocity);
 
     v[0][i] = - wdetJb * drho * u_normal;
     for (int j=0; j<3; j++)
-      v[j+1][i] = -wdetJb * (drho * u_normal * velocity[j] + norm[j] * dP);
+      v[j+1][i] = -wdetJb * (drho * u_normal * s.Y.velocity[j] + norm[j] * dP);
     v[4][i] = - wdetJb * u_normal * (dE + dP);
```

Comment:
Similar here for using `FluxTotal_Boundary`

Reply:
Nice, I did try to use `FluxTotal_Boundary()` but I ended up writing more code since the final state variable is coming from different sources. Can we merge this now and let me think about it more and change it when I add primitive variables?

Reply:
Yeah, that's fine!